docusaurus.config.js

```diff
@@ -56,6 +56,11 @@ const config = {
   themeConfig:
     /** @type {import('@docusaurus/preset-classic').ThemeConfig} */
     ({
+      colorMode: {
+        defaultMode: 'dark',
```

Comment:
i tried it out and while the switch is gone, it only shows the light version; Do you see the same issue?

Reply:
Hi @Siegfriedk, very estrange. Not to me, and the line 62:
"respectPrefersColorScheme:  false" is actually there to prevent any user preference set, [docusaurus](https://docusaurus.io/docs/api/themes/configuration#respectPrefersColorScheme)